crates/burn-import/src/burn/node/split.rs

```diff
@@ -0,0 +1,137 @@
+use super::{Node, NodeCodegen};
+use crate::burn::{OtherType, Scope, TensorType, ToTokens, Type};
+use burn::config::Config;
+use burn::record::PrecisionSettings;
+use proc_macro2::TokenStream;
+use quote::quote;
+
+#[derive(Config, Debug)]
+pub struct SplitConfig {
+    pub axis: usize,
+    pub num_outputs: Option<usize>,
+    pub split_sizes: Option<Vec<usize>>,
+}
+
+#[derive(Debug, Clone, new)]
+pub struct SplitNode {
+    pub input: TensorType,
+    pub outputs: Vec<TensorType>,
+    pub config: SplitConfig,
+}
+
+impl<PS: PrecisionSettings> NodeCodegen<PS> for SplitNode {
+    fn output_types(&self) -> Vec<Type> {
+        let tensor = &self.outputs[0];
+        let dims = tensor.dim;
+
+        let dims_literal = proc_macro2::Literal::usize_unsuffixed(dims);
+
+        let vec_tensor_type = quote! {Vec<Tensor<B, #dims_literal>>};
+
+        let other_type = OtherType {
+            name: syn::Ident::new("split_tensors", proc_macro2::Span::call_site()),
+            ty: vec_tensor_type,
+        };
+
+        vec![Type::Other(other_type)]
```

Comment:
Fixed number of outputs based on the parsed node outputs

```suggestion
        self.outputs
            .iter()
            .map(|t| Type::Tensor(t.clone()))
            .collect()
```